src/main/scala/com/fulcrumgenomics/sopt/cmdline/ClpArgumentDefinitionPrinting.scala

```diff
@@ -93,24 +96,22 @@ object ClpArgumentDefinitionPrinting {
   private def makeCollectionArity(argumentDefinition: ClpArgument): Option[String] = {
     if (!argumentDefinition.isCollection) return None
 
-    val desciption = (argumentDefinition.minElements, argumentDefinition.maxElements) match {
+    val description = (argumentDefinition.minElements, argumentDefinition.maxElements) match {
       case (0, Integer.MAX_VALUE) => "*"
       case (1, Integer.MAX_VALUE) => "+"
-      case (m, n)                 => s"{${m}..${n}}"
+      case (m, n)                 => s"{$m..$n}"
     }
-    Some(desciption)
+    Some(description)
   }
 
   /**
-    * Intelligently decides whether or not to print a default value. Values are not printed if
-    *  a) There is no default
-    *  b) There is a default, but it is 'None'
-    *  c) There is a default, but it's an empty list
-    *  d) There is a default, but it's an empty set
+    * Prints the default value if available, otherwise a message that it is not required.  This assumes the argument is
+    * optional.
     */
   private[sopt] def makeDefaultValueString(value : Option[_]) : String = {
     val vs = defaultValuesAsSeq(value)
-    if (vs.isEmpty) "" else s"[Default: ${vs.mkString(", ")}]."
+    // NB: extra square brackets are inserted due to one set being stripped during markdown processing
+    KGRN(if (vs.isEmpty) "[[Not required]]." else s"[[$ArgumentDefaultValuePrefix ${vs.mkString(", ")}]].")
```

Comment:
Two thoughts:

1. I'm not entirely sure that this catches all cases correctly?  Hopefully the tests will show me, but does it work correctly for e.g. required arguments with default values or optional values with None/Empty Seq etc.?
2. Could we change `Not required` to `Optional` please?

Reply:
The method assumes that the argument is optional (`Seq`, `Option`, or has a default value).

1. There are tests in `ClpArgumentDefinitionPrintingTest`.